opre_ops/ops_site/migrations/0007_canamount_caninfo.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.2.6 on 2021-09-17 21:45
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ops_site', '0006_alter_person_division'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CANInfo',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('number', models.CharField(max_length=30)),
+                ('description', models.CharField(max_length=100)),
+                ('nickname', models.CharField(max_length=30)),
+                ('purpose', models.TextField(null=True)),
+                ('arrangement_type', models.CharField(choices=[('1', 'OPRE Appropriation'), ('2', 'Cost Share'), ('3', 'IAA'), ('4', 'IDDA'), ('5', 'MOU')], max_length=30)),
+                ('authorizer', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, related_name='authorizer', to='ops_site.agency')),
```

Comment:
[_praise_] The Google Sheet describing this table says "On_delete(SET_NULL) or (PROTECT)". I think Protect was the right choice here! In general I see us using soft delete strategies, where we retain records in the database and store some state like "active"/"inactive" rather than hard-deleting database entries. I like the idea of an error being raised if an operation tries to delete an agency referenced by CAN data.